### DISCUSSION TITLE
tasmota32-webcam No 'buzzer' command?

### DISCUSSION THREAD

**Discussion by MaceMoneta** [arendst/Tasmota#15892] [Q&A] [answered]
I'm using the 12.0.2 release build of tasmota32-webcam on an AI Thinker board. I tried to use the 'buzzer' command, but get:

**13:50:22.676 CMD: buzzer
13:50:22.685 RSL: RESULT = {"Command":"Unknown"}**

I didn't see anything in the documentation indicating that it is an esp8266-only command.

Edit: Nevermind, I see from the code:

#ifdef ESP32
      analogAttach(Pin(GPIO_BUZZER), 7);
#endif  // ESP32

So, wrong pin.

**Comment by sfromis** [comment, marked as answer]
Not "wrong pin", but you need to configure a Buzzer pin before you can buzz

**Comment by MaceMoneta** [comment]
Thanks!
